README.md

```diff
@@ -92,22 +92,42 @@ To use the Material theme, import the correspondent file from the `theme/materia
   `vaadin-element.html`
 
 
-## Running demos and tests in browser
+## Running demos and tests in a browser
+Ensure you have [npm](https://www.npmjs.com/) and [Bower](https://bower.io) installed.
 
-1. Fork the `vaadin-element` repository and clone it locally.
+Clone the `vaadin-select` repository.
+```bash
+$ git clone https://github.com/vaadin/vaadin-select.git
+```
+
+Change into the `vaadin-select` directory.
+```bash
+$ cd vaadin-select
+```
+
+Install the dependencies.
+```bash
+$ npm install
```

Comment:
Same as above, one line is probably enough:

```sh
npm install && bower install
```

Reply:
It is clearer and easier for people to understand commands when there is only one command per line. There is no benefit to combining commands as space in the document is not an issue.

Clarity is better than obfuscation.